manifests/init.pp

```diff
@@ -245,6 +246,7 @@
       fastcgi_cache_use_stale        => $fastcgi_cache_use_stale,
       gzip                           => $gzip,
       http_access_log                => $http_access_log,
+      http_cfg_prepend                => $http_cfg_prepend,
```

Comment:
This is causing the Travis tests to fail, because the `=>` is not aligned with the rest. Puppet Lint is running in Travis and checks for this.